doc-content/enterprise-only/admin-and-config/logging-configuration-logback-proc.adoc

```diff
@@ -0,0 +1,17 @@
+[id='logging-configuration-logback-proc']
+= Logging configurations using `logback`
+
+You can use `logback` for logging configuration. The configurations are logged to the Simple Logging Facade for Java SJF4J, which delegates the configuration log to `logback`.
+
+.Procedure
+. Add a dependency to the logging adaptor for your logging framework.
+. If you are not using a logging framework, use `logback` by adding the following Maven dependency.
++
+[source,code]
+----
+<dependency>
+   <groupId>ch.qos.logback</groupId>
+   <artifactId>logback-classic</artifactId>
+   <version>1.x</version>
+</dependency>
```

Comment:
Same here, this cannot be configured for Business Central or KIE Server.

Reply:
Moving this section to process engine doc